docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -94,6 +94,12 @@ The component takes care of the most used properties, then it's up to the user t
 
 {{"demo": "pages/components/text-fields/Inputs.js"}}
 
+## Color
+
+The `color` prop changes the highlight color of the text field.
```

Comment:
Should we be more specific, to the only support of the secondary value? At the same time, we should extend support in the near future. I would rather avoid having to update the description.

Reply:
I feel like this is something most people will check on the API-page anyways.